src/PlayerSummary.js

```diff
@@ -0,0 +1,54 @@
+import Affiliation from './Affiliations.js';
+import LoadOutInventory from './LoadOutInventory.js';
+import Mission from './Mission.js';
+import OperatorLoadOuts from './OperatorLoadOuts.js';
+import PlayerSkill from './PlayerSkill.js';
+import { parseDate } from './utils.js';
+
+export default class PlayerSummary {
+  constructor(summary) {
+    this.accountId = summary.AccountId.$oid;
+    this.displayName = summary.DisplayName;
+    this.playLevel = summary.PlayerLevel;
+    this.loadout = new LoadOutInventory(summary.LoadOutInventory);
+    this.playerSkills = new PlayerSkill(summary.PlayerSkills);
+
+    // TODO: See if the nightwave challenges use the same name in worldstate-data then translate
+    this.challengeProgress = summary.ChallengeProgress;
+    this.guildId = summary.GuildId.$oid;
+    this.guildName = summary.GuildName;
+    this.guildTier = summary.GuildTier;
+    this.guildXp = summary.GuildXp;
+    this.guildClass = summary.GuildClass;
+    this.guildEmblem = summary.GuildEmblem;
+    this.allianceId = summary.AllianceId.$oid;
+    this.deathMarks = summary.DeathMarks;
+    this.harvestable = summary.Harvestable;
+
+    // TODO: might be able to translate this once we update enemy data.
+    this.deathSquadable = summary.DeathSquadable;
+    this.created = parseDate(summary.Created);
+    this.migratedToConsole = summary.MigratedToConsole;
+    this.missions = summary.Missions.map((m) => new Mission(m));
+    this.affiliations = summary.Affiliations.map((a) => new Affiliation(a));
+    this.dailyAffiliation = summary.DailyAffiliation;
+    this.dailyAffiliationPvp = summary.DailyAffiliationPvp;
+    this.dailyAffiliationLibrary = summary.DailyAffiliationLibrary;
+    this.dailyAffiliationCetus = summary.DailyAffiliationCetus;
+    this.dailyAffiliationQuills = summary.DailyAffiliationQuills;
+    this.dailyAffiliationSolaris = summary.DailyAffiliationSolaris;
+    this.dailyAffiliationVentkids = summary.DailyAffiliationVentkids;
+    this.dailyAffiliationVox = summary.DailyAffiliationVox;
+    this.dailyAffiliationEntrati = summary.DailyAffiliationEntrati;
+    this.dailyAffiliationNecraloid = summary.DailyAffiliationNecraloid;
+    this.dailyAffiliationZariman = summary.DailyAffiliationZariman;
+    this.dailyAffiliationKahl = summary.DailyAffiliationKahl;
+    this.dailyAffiliationCavia = summary.DailyAffiliationCavia;
+    this.dailyFocus = summary.DailyFocus;
+    this.wishList = summary.Wishlist;
+    this.unlockedOperator = summary.UnlockedOperator;
+    this.unlockedAlignment = summary.UnlockedAlignment;
+    this.operatorLoadouts = summary.OperatorLoadOuts.map((ol) => new OperatorLoadOuts(ol));
+    this.alignment = { wisdom: summary.Alignment.Wisdom, alignment: summary.Alignment.Wisdom };
```

Comment:
Seems like the same thing twice... Should it be something other than wisdom and wisdom?

Reply:
Copy paste error. I've been fixing small stuff like this as I've been adding docs. will be corrected in the next commit